static/js/unread.js

```diff
@@ -264,9 +264,55 @@ exports.num_unread_for_person = function (user_ids_string) {
     return unread_privates.get(user_ids_string).num_items();
 };
 
+exports.current_stream_topic = function () {
+    var selected_stream;
+    var selected_topic;
+    if (narrow.filter() === undefined || narrow.filter()._operators["1"] === undefined) {
+         var message_header = $(".selected_message").closest(".recipient_row").find(".message_header");
```

Comment:
We shouldn't be using jQuery here.  There should be a way to get the current message object using `message_store.js` or similar code, and then you can pull stream_id and topic more directly from the message.  We may also want to move this code to something like `narrow.js`, since this particular function has nothing to do with unread counts, and has more to do with how you are narrowed.  Pinging @timabbott for his thoughts on a) how to do this more efficiently, b) where to put this code, and c) are these the correct semantics for when you're in the home view?